sqlmesh/core/model/definition.py

```diff
@@ -1026,6 +1027,14 @@ def update_schema(
         super().update_schema(
             schema, default_schema=default_schema, default_catalog=default_catalog
         )
+
+        mocked_star = normalize_identifiers(SQLMESH_MOCKED_STAR, dialect=self.dialect)
```

Comment:
why do you need to normalize it? just have the mocked star be quoted?

Reply:
It doesn't work for all dialects - Redshift for example doesn't care about quotes.